cmd/node/main.go

```diff
@@ -509,9 +518,21 @@ func startNode(ctx *cli.Context, log *logger.Logger) error {
 
 	ef := facade.NewElrondNodeFacade(currentNode, externalResolver)
 
+	prometheusJoinURLAvailable := true
+	prometheusJoinUrl, err := getPrometheusJoinURL(ctx.GlobalString(serversConfigurationFile.Name))
+	if err != nil {
```

Comment:
can you do a combined conditional err!=nil || prometheusJoinUrl== ""  since these have the same outcome

Reply:
fixed